act-rules-format.md

```diff
@@ -206,6 +215,82 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
+
+Some scenarios when a rule will have Secondary requirements include:
+
+**Scenario 1**: only the "Failed Outcomes" are true for the requirement
+
+A rule was designed to test a minimum accessibility requirement and there exists a stricter requirement. In this scenario, the rule’s failed outcomes can determine that the stricter accessibility requirement is not satisfied, and the rule’s passed outcomes may not determine that the stricter requirement is satisfied. It is possible that the accessibility requirement may be not satisfied when the rules outcomes are passed. The stricter requirement is a Secondary requirement.
+
+<aside class=example>
+  <header>Example: a rule that tests if text has minimum contrast</header>
+<blockquote>This rule was designed to test Success Criterion 1.4.3 Contrast Minimum (AA). A stricter requirement, Success Criterion 1.4.6 Contrast (Enhanced) (Level AAA), will be not satisfied when the rule outcome is failed, and may be not satisfied when the rule outcomes are passed. This rule’s mapping:
+<ul>
+<li>Conformance Requirement: Success Criterion 1.4.3 Contrast (Minimum)</li>
+<li>Secondary Requirement: Success Criterion 1.4.6 Contrast (Enhanced)</li>
+  <ul><li>Background Section: Success Criterion 1.4.6 is mapped as a Secondary requirement because the SC may be <dfn>not satisfied</dfn>when the rule’s outcomes are `passed`. </li></ul>
+</ul>
+</blockquote>
+</aside>
+
+**Scenario 2**: only the "Passed or Inapplicable Outcomes" are true for the requirement
+<ol style="list-style-type: lower-alpha">
+  <li>A rule was designed to test a specific type of solution for an accessibility requirement, but the rule does not cover all solutions that can be used to meet the requirement. In this scenario, the rule’s failed outcomes cannot determine that an accessibility requirement is not satisfied because further testing is needed. The rule’s passed outcomes can determine that an accessibility requirement is satisfied. The accessibility requirement is a Secondary requirement.</li>
+
+  <aside class=example>
+    <header>Example: a rule that tests if a focusable element has no keyboard trap via standard navigation</header>
+    <blockquote>This rule was designed to test for a specific solution that can be used to meet Success Criterion 2.1.2: No Keyboard Trap. The rule does not test for other possible solutions, such as the use of non-standard navigation, so its failed outcomes cannot determine that the accessibility requirement is <dfn>not satisfied</dfn>. This rule’s mapping:
+<ul>
+<li>Secondary Requirement: Success Criterion 2.1.2: No Keyboard Trap</li>
+  <ul><li>Background Section: This rule tests for one solution that can meet Success Criterion 2.1.2. SC 2.1.2 is mapped as a Secondary requirement because other solutions (such as use of non-standard navigation to exit a keyboard trap) can be used to meet this SC that are not tested by this rule.</li></ul>
```

Comment:
Here also, the "further testing is needed" bit in the "Passed or Inapplicable Outcomes" condition make this too much of a "catch all" case.

For example, when all outcomes of this rule ar `passed` or `inapplicable`, further testing is needed to decide whether the test subject satisfy 2.4.2 Page Titled (because we essentially haven't looked at it yet…)
But then, for this rule and SC 2.4.2, one of the condition is true and the other false, which means it should be a secondary requirement 🤔